addon/components/table.ts

```diff
@@ -383,39 +473,29 @@ class TableComponent extends Component.extend({ ResizeAware }) {
      * @memberof TableComponent
      */
     updateColumnVisibility() {
-        const columns = this.columns || A();
-        const visibleColumns: NativeArray<TableColumn> = A();
-        const containerWidth = this.getElementWidth(this.element);
+        const columns = this.args.columns || A();
+        const visibleColumns: NativeArray<ColumnValue> = A();
+        const containerWidth = this.getElementWidth(window.document.querySelector(`#${this.elementId}`)!);
```

Comment:
we should probably store the container element once in the `didInsertTable()` call, and you can use `getElementById` here as it'll be a little faster:
```ts
didInsertTable() {
    this.containerElement = document.getElementById(this.elementId);
    //...
}
```